scripts/cnv_cromwell_tests/somatic/cnv_somatic_panel_wgs_no-gc_workflow.json

```diff
@@ -1,7 +1,10 @@
 {
   "CNVSomaticPanelWorkflow.gatk_docker": "__GATK_DOCKER__",
   "CNVSomaticPanelWorkflow.intervals": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/chr20.interval_list",
-  "CNVSomaticPanelWorkflow.normal_bams_list": "/home/travis/build/broadinstitute/gatk/scripts/cnv_cromwell_tests/somatic/cnv_somatic_panel_wgs_input.tsv",
+  "CNVSomaticPanelWorkflow.normal_bams": ["/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam",
```

Comment:
bais before bams to maintain sort order.

Reply:
Done